menas/ui/components/AuditTrail.js

```diff
@@ -24,10 +24,10 @@ sap.ui.define(["components/tables/TableUtils"],
 
   AuditTrail.prototype.applyTableUtils = function() {
     const auditTableUtils = new TableUtils(this._oAuditTable, "Audit Trail");
+    auditTableUtils.makeSearchable(["updatedBy", "changes"]);
```

Comment:
Just curious if there is any reason for moving this?

Will test the FrontEnd later. Code seems fine.

Reply:
Yeah there's no explicit ordering of the items in the toolbar. So basically want the search bar to appear before group and filter buttons.